pkgs/by-name/do/dosbox-staging/package.nix

```
@@ -90,7 +81,10 @@ stdenv.mkDerivation (finalAttrs: {
       ]
     );
 
-  outputs = [ "out" "man" ];
+  outputs = [
+    "out"
+    "man"
+  ];
 
   postInstall = ''
     install -Dm644 $src/contrib/linux/dosbox-staging.desktop $out/share/applications/
```

```suggestion
    install -Dm644 $src/contrib/linux/org.dosbox-staging.dosbox-staging.desktop $out/share/applications/
```
Desktop file was changed to this. https://github.com/dosbox-staging/dosbox-staging/blob/main/contrib/linux/org.dosbox-staging.dosbox-staging.desktop